measures/BuildExistingModel/measure.rb

```diff
@@ -361,7 +362,6 @@ def run(model, runner, user_arguments)
     # Set arguments for the BuildResidentialHPXML measure
     hpxml_path = File.expand_path('../existing.xml')
     measures['BuildResidentialHPXML'] = [{ 'hpxml_path' => hpxml_path }]
-    measures['BuildResidentialHPXML'][0]['apply_defaults'] = true # for apply_hvac_sizing since ApplyUpgrade sets HVAC capacities
     measures['BuildResidentialHPXML'][0]['apply_validation'] = true
```

Comment:
Maybe there is no point in doing the validation here anymore. Instead, should we do the validation (twice? i.e., before applying defaults _and_ at the end) in the PostHPXML measure? Or re-enable validation in the HPXMLtoOS measure and do validation once (before applying defaults) in the PostHPXML measure?